src/integrationTest/java/uk/gov/hmcts/reform/sendletter/tasks/StaleLettersTaskTest.java

```diff
@@ -0,0 +1,141 @@
+package uk.gov.hmcts.reform.sendletter.tasks;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sendletter.entity.Letter;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.entity.LetterStatus;
+import uk.gov.hmcts.reform.sendletter.logging.AppInsights;
+import uk.gov.hmcts.reform.sendletter.services.FtpAvailabilityChecker;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.sendletter.util.MessageIdProvider.randomMessageId;
+
+@RunWith(SpringRunner.class)
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+public class StaleLettersTaskTest {
+
+    @Autowired
+    private LetterRepository repository;
+
+    private LocalTime cutOff;
+
+    private LocalTime secondBeforeCutOff;
+
+    @Mock
+    private AppInsights insights;
+
+    private StaleLettersTask task;
+
+    @Before
+    public void setUp() {
+        FtpAvailabilityChecker checker = new FtpAvailabilityChecker("13:00", "14:00");
+        task = new StaleLettersTask(repository, insights, checker);
+        cutOff = checker.getDowntimeStart();
+        secondBeforeCutOff = cutOff.minusSeconds(1);
+
+        // making sure each test has fresh table
+        repository.deleteAll();
+    }
+
+    @Test
+    public void should_do_nothing_when_there_are_no_unprinted_letters() {
+        // when
+        task.run();
+
+        // then
+        verify(insights, never()).trackStaleLetter(any(Letter.class));
+    }
+
+    @Test
+    @SuppressWarnings("VariableDeclarationUsageDistance")
+    public void should_report_to_insights_when_there_is_an_unprinted_letter() {
+        // given
+        Letter letter = createLetter(secondBeforeCutOff);
```

Comment:
looks like it's the date when letter was created, but it's when it was sent to print. worth updating method name imgo